net.hhoney.tinycrate.yml

```diff
@@ -0,0 +1,30 @@
+id: net.hhoney.tinycrate
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.godotengine.godot.BaseApp
+base-version: '3.6'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=pulseaudio
+- --device=all
+modules:
+- name: tinycrate
+  buildsystem: simple
+  sources:
+
+  - type: git
+    url: https://github.com/HarmonyHoney/tiny_crate.git
+    commit: 8adf13fd943e958366c04cd12287b8677ae145ca
+
+  - type: file
+    url: https://github.com/HarmonyHoney/tiny_crate/releases/download/2024.12.16/Tiny-Crate.pck
+    sha256: 538cd6dded643b93011fbd6681da2dbe5f5d01c8d56625d4c1cc77dce174fd29
+
+  build-commands:
+  - install -Dm644 Tiny-Crate.pck ${FLATPAK_DEST}/bin/godot-runner.pck
+  - install -Dm644 linux/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+  - install -Dm644 media/image/android_icon.png ${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/${FLATPAK_ID}.png
```

Comment:
this is not a valid icon size. Likely why the build failed.

Reply:
Yeah that's my assumption as well. (I feel like it was a size we used in elementary and it doesn't say there are certain sizes it has to be, just _larger_ than 128px… but I digress…)